apps/backoffice-v2/src/domains/business-reports/hooks/mutations/useCreateBusinessReportBatchMutation/useCreateBusinessReportBatchMutation.tsx

```diff
@@ -0,0 +1,45 @@
+import { t } from 'i18next';
+import { toast } from 'sonner';
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+
+import { HttpError } from '@/common/errors/http-error';
+import { TBusinessReportType } from '@/domains/business-reports/types';
+import { createBatchBusinessReport } from '@/domains/business-reports/fetchers';
+import { useCustomerQuery } from '@/domains/customer/hook/queries/useCustomerQuery/useCustomerQuery';
+
+export const useCreateBusinessReportBatchMutation = ({
```

Comment:
Would make sense for the naming to be consistent with the fetcher createBatchBusinessReport vs createBusinessReportBatch